### DISCUSSION TITLE
Support for non-Claimable wishes

### DISCUSSION THREAD

**Discussion by tpansino** [cmintey/wishlist#185] [Ideas]
```
As the owner of a wishlist/registry
I want to be able to display some wishlist items without the Claim button
So that links for things like gift cards, or links to other wishlists (Amazon, Steam) can be permanently listed, and not be accidentally removed because someone marked them as claimed.
```

I can think of two ways one might implement this:
1. On the wish creation screen, add a toggle for whether the wish should be Claimable/Unclaimable, and show/hide the Claim button on the wishlist appropriately
2. On the wish creation screen, add a Quantity counter. Then allow users to make multiple claims on that wish, up to the saved Quantity. You could then store a quantity of -1 or something for an Unclaimable item (not sure what the UI for that would look like though).

**Comment by cmintey** [comment]
Hm, that's interesting. I have thought about adding a "quantity" to items. If that were the case, then there could be an "infinite" option. With that option, I think it would still make sense to be able to claim it so it would be added to your claims page. But since it's "infinite", it doesn't show as claimed for others so they can also claim it. Thoughts on that?

**Reply by tpansino** [reply]
I haven't used the Claims page much myself, but yes that implementation does sound like it would cover multiple use cases.
